production/technical_userguide.md

```diff
@@ -666,3 +666,66 @@ curl https://api.bcda.cms.gov/data/45/99dbc86f-74e5-4553-88a9-af3e718cb72b.ndjso
 The response will be the requested data as [FHIR Coverage resources](https://www.hl7.org/fhir/coverage.html){:target="_blank"} in NDJSON format.
 
 An example of one such resource is available in the [guide to working with BCDA data](/data-guide/#sample-bcda-files).
+
+### Filtering Your Data with `_since`
+#### About `_since` 
+The `_since` parameter grants you the ability to apply a date parameter to your bulk data requests. Instead of receiving the full record of historical data every time you request data from an endpoint, you will be able to use `_since` to submit a date. BCDA will then produce claims data from the bulk data endpoints that have been loaded since the entered date.
+
+For more information on `_since`, please consult the [FHIR standard on query parameters](https://hl7.org/Fhir/uv/bulkdata/export/index.html#query-parameters).
```

Comment:
Can you apply a `{:target="_blank"}` to this link so it opens in a new tab (since it is not related to the `cms.gov` domain)?  There are other examples in this file that you can use as examples.

Reply:
Changed, resolved!